apps/odo/.gitignore

```diff
@@ -0,0 +1,30 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
i think we can safely remove this sub-gitignore

Reply:
Removed